### PR TITLE
docs: replace pgpolicy with crudpolicy

### PR DESCRIPTION
**Pull request by dhanushreddy291**
- Replace `pgPolicy` with `crudPolicy`

preview:
- https://neon-next-git-fork-dhanushreddy291-update-d-649b06-neondatabase.vercel.app/docs/guides/neon-authorize-clerk#1-add-row-level-security-policies
- https://neon-next-git-fork-dhanushreddy291-update-d-649b06-neondatabase.vercel.app/docs/guides/neon-authorize-stack-auth#1-add-row-level-security-policies
- https://neon-next-git-fork-dhanushreddy291-update-d-649b06-neondatabase.vercel.app/docs/guides/neon-authorize-stack-auth#1-add-row-level-security-policies

### CODE REVIEW

File: content/docs/guides/neon-authorize-clerk.md [hunk 1]
```diff
@@ -257,51 +263,50 @@ export async function TodoList() {
 ```typescript shouldWrap
 'use client';
 
-  import type { Todo } from '@/app/schema';
```
Comment (dhanushreddy291) [inline content/docs/guides/neon-authorize-clerk.md:260]: all of this were just proper indentation changes

File: content/docs/guides/neon-authorize-auth0.md [hunk 2]
```diff
@@ -141,16 +141,18 @@ Now that you’ve integrated Auth0 with Neon Authorize, you can securely pass JW
 
 ### 1. Add Row-Level Security policies
 
-Below are examples of RLS policies for a **todos** table, designed to restrict access so that users can only create, view, update, or delete their own todos.
+Here's how to implement RLS policies for a **todos** table using a `crudPolicy`. This policy ensure users can only access and modify their own todos.
```
Comment (bgrenon) [inline content/docs/guides/neon-authorize-auth0.md:144]: Let's keep the language explaining these are examples only. We don't want people thinking they need to copy-paste these samples as part of the mandatory setup. Also, the drizzle tab is using crudPolicy, but the SQL tab is still using granular RLS.

"Here are examples of implementing RLS policies for a **todos** table – the Drizzle example leverages the simplified `crudPolicy` function, while the SQL example demonstrates the use of individual RLS policies."

**Comment by bgrenon** [comment]
@davidgomes can you review the change to crudPolicy for the Drizzle tabs on these quickstart pages?

File: content/docs/guides/neon-authorize-auth0.md (hunk 2 above)
Comment (dhanushreddy291) [reply content/docs/guides/neon-authorize-auth0.md]: updated

File: content/docs/guides/neon-authorize-auth0.md [hunk 3]
```diff
@@ -141,16 +141,18 @@ Now that you’ve integrated Auth0 with Neon Authorize, you can securely pass JW
 
 ### 1. Add Row-Level Security policies
 
-Below are examples of RLS policies for a **todos** table, designed to restrict access so that users can only create, view, update, or delete their own todos.
+Here are examples of implementing RLS policies for a todos table – the Drizzle example leverages the simplified crudPolicy function, while the SQL example demonstrates the use of individual RLS policies.
```
Comment (bgrenon) [inline content/docs/guides/neon-authorize-auth0.md:144]:
```suggestion
Here are examples of implementing RLS policies for a **todos** table – the Drizzle example leverages the simplified `crudPolicy` function, while the SQL example demonstrates the use of individual RLS policies.
```

File: content/docs/guides/neon-authorize-clerk.md [hunk 4]
```diff
@@ -123,15 +144,16 @@ Now that you’ve integrated Clerk with Neon Authorize, you can securely pass JW
 
 ### 1. Add Row-Level Security policies
 
-Here are some examples of RLS policies for a **todos** table, designed to restrict access so that users can only create, view, update, or delete their own todos.
+Here are examples of implementing RLS policies for a todos table – the Drizzle example leverages the simplified crudPolicy function, while the SQL example demonstrates the use of individual RLS policies.
```
Comment (bgrenon) [inline content/docs/guides/neon-authorize-clerk.md:147]:
```suggestion
Here are examples of implementing RLS policies for a **todos** table – the Drizzle example leverages the simplified `crudPolicy` function, while the SQL example demonstrates the use of individual RLS policies.
```

**Review by bgrenon — APPROVED** [review]

File: content/docs/guides/neon-authorize-stack-auth.md [hunk 5]
```diff
@@ -120,16 +141,18 @@ Now that you’ve integrated Stack Auth with Neon Authorize, you can securely pa
 
 ### 1. Add Row-Level Security policies
 
-Below are examples of RLS policies for a **todos** table, designed to restrict access so that users can only create, view, update, or delete their own todos.
+Here are examples of implementing RLS policies for a todos table – the Drizzle example leverages the simplified crudPolicy function, while the SQL example demonstrates the use of individual RLS policies.
```
Comment (bgrenon) [inline content/docs/guides/neon-authorize-stack-auth.md:144]:
```suggestion
Here are examples of implementing RLS policies for a **todos** table – the Drizzle example leverages the simplified `crudPolicy` function, while the SQL example demonstrates the use of individual RLS policies.
```

**Pull request merged**